common/types.ts

```diff
@@ -19,6 +19,12 @@ export interface User {
   isAdmin: boolean;
   rsvpStatus: RSVPStatus;
 }
+export interface SingletonInfo {
+  type: string;
```

Comment:
I don't like this, I was thinking about just having this be 
`type: string` and then one instance would just be like `type: 'registration-open'` and we could just search the DB by type name - does require string inputs tho which is less than ideal